tyche-update-helper.php

```diff
@@ -3,15 +3,16 @@
 Plugin Name: Tyche Update Helper
 Plugin URI: #
```

Comment:
Please add plugin URI to https://www.tychesoftwares.com/premium-woocommerce-plugins/